flake.nix

```diff
@@ -20,10 +20,10 @@
           name = "http4s";
           typelevelShell = {
             jdk.package = jdk;
-            nodejs.enable = false;
+            nodejs.enable = true;
```

Comment:
I don't know how this worked before.  It was probably using an ambient node from the container?  ("Pure" development shells stop this, but we'd have more work to do...)

Reply:
Oh, never mind, you had moved it for the node 16 workaround.